lib/src/database/database_file_storage_stub.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:typed_data';
+
+mixin DatabaseFileStorage {
+  bool get supportsFileStoring => fileStoragePath != null;
```

Comment:
Using this stup is must always be `false`.

Reply:
Would be better, yes. Will change it